src/Navigator.jl

```diff
@@ -97,20 +98,21 @@ function comp_weights(navabs::Array{T, 4}, noisestd::Matrix{T}, lines::Int64, sl
 
     # weights[points, coils, lines, slices]
     coils = size(navabs, 2)
-    weights = zeros(size(navabs))
-    for ii=1:coils
-        weights[:,ii,:,:] = navabs[:,ii,:,:] ./ noisestd[1,ii]
-    end
+    weights = ones(size(navabs))
+    if !any(noisestd .== 0)
```

Comment:
Do we know for sure an explicit value of 0 is always given? I.e it couldn't be 1e-31 or something by accident?